test/cpp/interop/istio_echo_server.cc

```diff
@@ -153,6 +149,7 @@ void RunServer(std::vector<int> grpc_ports, std::set<int> xds_ports,
 }  // namespace grpc
 
 int main(int argc, char** argv) {
+  std::this_thread::sleep_for(std::chrono::seconds(30));
```

Comment:
why?

Reply:
this was originally to fix a case where the istio agent had not yet prepared the bootstrap file but https://github.com/istio/istio/pull/39507 should fix it I believe